Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 import MapboxCoreNavigation
 import MapboxDirections
 
-public protocol TopBannerViewControllerDelegate: AnyObject, UnimplementedLogging {
+public protocol TopBannerViewControllerDelegate: VisualInstructionDelegate {
```

Comment:
Keep `AnyObject` and `UnimplementedLogging` here too.

Reply:
Same as the `InstructionsBannerViewDelegate`. There's a redundant declaration of  `AnyObject` and `UnimplementedLogging` because `VisualInstructionDelegate` already conforms to them.